source/install/cloud-changelog.md

```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-05-04](#release-2023-05-04)
 - [Release 2023-04-21](#release-2023-04-21)
 - [Release 2023-03-29](#release-2023-03-29)
 - [Release 2023-03-20](#release-2023-03-20)
 - [Release 2023-01-26](#release-2023-01-26)
 - [Release 2023-01-16](#release-2023-01-16)
-- [Release 2022-12-20](#release-2022-12-20)
+
+## Release 2023-05-04
+
+### Highlights
+ - Added an improved Cloud trial experience in preparation for replacing the Cloud Free experience. See more details in these Forum posts:
+    - https://forum.mattermost.com/t/cloud-monthly-billing-deprecation/15804
+    - https://forum.mattermost.com/t/cloud-free-deprecation/15802
+
+### Improvements
+
+#### User Interface (UI)
+ - Replaced the ``compass-components`` icon component with ``compass-icons``.
+ - Added “hours ahead” timezone details to the user profile popover.
+ - Pre-packaged Calls v0.15.1.
+
+#### Administration
+ - Removed the deprecated ``model.CommandArgs.Session``.
+ - The database section in the **System Console** now has an additional read-only section which shows the active search backend in use. This can be helpful to confirm which search engine is currently active when there are multiple of them configured.
```

Comment:
```suggestion
 - The database section in the **System Console** now has an additional read-only section which shows the active search backend in use. This can be helpful to confirm which search engine is currently active when there are multiple configured.
```